functions/handlers/books.js

```diff
@@ -52,6 +53,7 @@ exports.postOneBook = (req, res) => {
     })
     .catch((err) => {
       res.status(500).json({ error: "Something went wrong :(" });
+      res.json(err);
```

Comment:
we already have  
`res.status(500).json({ error: "Something went wrong :(" });` 

so the response given will be the object { error: "Something went wrong :(" } not err. If you want to return the error object instead change the above line to 
` res.status(500).json(err)`

Reply:
For sure, I will change it.